### ISSUE TITLE
Date picker component accessibility issues

### ISSUE THREAD

**Issue by rohitbhoite93** [SAP/fundamental-ngx#7903]
#### Is this a bug, enhancement, or feature request?
Accessibility bug
#### Briefly describe your proposal.
These issues are observed in Date picker component on Android mobile app which uses NGX library. Accessibility tool used is talkback on android phone.

Current Behaviour: 
Issue 1: On accessing the weekdays S, M, T, W, T, F, S, weekdays are not read in long form. Also, row, column count is read. Speech announcement made as “Capital S column header row 1 in table 7 rows 7 columns”.
Issue 2: On accessing the calendar control, incorrect UI type “banner table” is read.
<img width="257" alt="image" src="https://user-images.githubusercontent.com/101723690/159290101-daaba847-278c-42bc-a6f1-e22de80c2da1.png">

Expected behaviour : 
Issue 1 : The weekdays Sunday, Monday, Tuesday and so on should be read out in elaborated form.
Issue 2:  When the calendar button is accessed, correct UI type should be read.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular :11.2.14 NGX library version : 0.32.1

**Comment by mikerodonnell89** [comment]
@rohitbhoite93 We are now displaying short form weekday names and they are being read correctly with Chrome screenreader (I am unable to test on android). But for issue 2, I'm hearing "grid" when I click this button, I think we can expand upon that a bit, what is the "correct UI type" you are hoping to hear?
